x-pack/plugins/maps/public/ems_autosuggest/ems_autosuggest.ts

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FileLayer } from '@elastic/ems-client';
+import { getEmsFileLayers, fetchGeoJson } from '../util';
+import { FORMAT_TYPE } from '../../common';
+
+export interface ISampleValuesConfig {
+  emsLayerIds?: string[];
+  sampleValues?: Array<string | number>;
+  sampleValuesColumnName?: string;
+}
+
+export interface IEMSTermJoinConfig {
+  layerId: string;
+  field: string;
+}
+
+const emsWorldLayerId = 'world_countries';
```

Comment:
Can you move these to constants and then replace all 'world_countries' with the constant. 'world_countries' is currently being used in 5 other places